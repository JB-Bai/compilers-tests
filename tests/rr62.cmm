int IGDVmVib(float WtcMZvYm[3462]){
	int Q1IS0fkxA[0xdF][000043] = oI8C5zvF() = aPeN9Dv7, KbXNWwuw8[0252][014];
	struct{
	}
	jtFG = 052520;
	struct WyVjH niLctcGA[0730];
	while(- 0x59.L1p_JWy >(Ffw72C) > tkBTloVs4O / SQeBjlR) 70.0e-7 - 3.72 - Axgmb() - 0550664[SRSkHtU4] || 1736.254E+3 / 0465 || 25.58e-1 + 647.191 * hePrIfd && 58.0 = vASMTA0a - 01.30 * aXhNmU_[fv_AsCUgzY || 32.886e+8 >= ! - IyLpM][8.008] && 041633;
	{
		(2486) ||(qcXQ_R5) ==(! _dPwd() / 036077.SuPihyf) / utLNtLS() / 00774 * 25010 * ! ktrVx5((2730.18)) = 4836.7763E9[yiozwUb9Ym] || t8BbRFZ0 / kwnAoY8;
		N_m7wo0d;
	}
	! - ZqLP = g7e52kv - 84.5377e-7;
}
struct OM7yrgCX{
	float AcbtgCZ[0x6DE4];
	float UMo7LK8uH[05374] = lfky7CXb7h8(Gc_P51G(85.06));
	float QOZ7u1F;
}
dicB5v8AH[01327], BOM6, l0iaji, JrnzCC4G8d, LJ99NK6Y, cDTCz[0x4c7][0x3], AMXKE[0xcd7];
int NAOUVsQOD;

