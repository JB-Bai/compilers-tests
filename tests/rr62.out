Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      FunDec (1)
        ID: IGDVmVib
        LP
        VarList (1)
          ParamDec (1)
            Specifier (1)
              TYPE: float
            VarDec (1)
              VarDec (1)
                ID: WtcMZvYm
              LB
              INT: 3462
              RB
        RP
      CompSt (1)
        LC
        DefList (2)
          Def (2)
            Specifier (2)
              TYPE: int
            DecList (2)
              Dec (2)
                VarDec (2)
                  VarDec (2)
                    VarDec (2)
                      ID: Q1IS0fkxA
                    LB
                    INT: 223
                    RB
                  LB
                  INT: 35
                  RB
                ASSIGNOP
                Exp (2)
                  Exp (2)
                    ID: oI8C5zvF
                    LP
                    RP
                  ASSIGNOP
                  Exp (2)
                    ID: aPeN9Dv7
              COMMA
              DecList (2)
                Dec (2)
                  VarDec (2)
                    VarDec (2)
                      VarDec (2)
                        ID: KbXNWwuw8
                      LB
                      INT: 170
                      RB
                    LB
                    INT: 12
                    RB
            SEMI
          DefList (3)
            Def (3)
              Specifier (3)
                StructSpecifier (3)
                  STRUCT
                  LC
                  RC
              DecList (5)
                Dec (5)
                  VarDec (5)
                    ID: jtFG
                  ASSIGNOP
                  Exp (5)
                    INT: 21840
              SEMI
            DefList (6)
              Def (6)
                Specifier (6)
                  StructSpecifier (6)
                    STRUCT
                    Tag (6)
                      ID: WyVjH
                DecList (6)
                  Dec (6)
                    VarDec (6)
                      VarDec (6)
                        ID: niLctcGA
                      LB
                      INT: 472
                      RB
                SEMI
        StmtList (7)
          Stmt (7)
            WHILE
            LP
            Exp (7)
              Exp (7)
                Exp (7)
                  MINUS
                  Exp (7)
                    Exp (7)
                      INT: 89
                    DOT
                    ID: L1p_JWy
                RELOP
                Exp (7)
                  LP
                  Exp (7)
                    ID: Ffw72C
                  RP
              RELOP
              Exp (7)
                Exp (7)
                  ID: tkBTloVs4O
                DIV
                Exp (7)
                  ID: SQeBjlR
            RP
            Stmt (7)
              Exp (7)
                Exp (7)
                  Exp (7)
                    Exp (7)
                      Exp (7)
                        Exp (7)
                          Exp (7)
                            FLOAT: 0.000007
                          MINUS
                          Exp (7)
                            FLOAT: 3.720000
                        MINUS
                        Exp (7)
                          ID: Axgmb
                          LP
                          RP
                      MINUS
                      Exp (7)
                        Exp (7)
                          INT: 184756
                        LB
                        Exp (7)
                          ID: SRSkHtU4
                        RB
                    OR
                    Exp (7)
                      Exp (7)
                        FLOAT: 1736254.000000
                      DIV
                      Exp (7)
                        INT: 309
                  OR
                  Exp (7)
                    Exp (7)
                      Exp (7)
                        FLOAT: 2.558000
                      PLUS
                      Exp (7)
                        Exp (7)
                          FLOAT: 647.190979
                        STAR
                        Exp (7)
                          ID: hePrIfd
                    AND
                    Exp (7)
                      FLOAT: 58.000000
                ASSIGNOP
                Exp (7)
                  Exp (7)
                    Exp (7)
                      ID: vASMTA0a
                    MINUS
                    Exp (7)
                      Exp (7)
                        FLOAT: 1.300000
                      STAR
                      Exp (7)
                        Exp (7)
                          Exp (7)
                            ID: aXhNmU_
                          LB
                          Exp (7)
                            Exp (7)
                              ID: fv_AsCUgzY
                            OR
                            Exp (7)
                              Exp (7)
                                FLOAT: 3288600064.000000
                              RELOP
                              Exp (7)
                                NOT
                                Exp (7)
                                  MINUS
                                  Exp (7)
                                    ID: IyLpM
                          RB
                        LB
                        Exp (7)
                          FLOAT: 8.008000
                        RB
                  AND
                  Exp (7)
                    INT: 17307
              SEMI
          StmtList (8)
            Stmt (8)
              CompSt (8)
                LC
                StmtList (9)
                  Stmt (9)
                    Exp (9)
                      Exp (9)
                        Exp (9)
                          LP
                          Exp (9)
                            INT: 2486
                          RP
                        OR
                        Exp (9)
                          Exp (9)
                            LP
                            Exp (9)
                              ID: qcXQ_R5
                            RP
                          RELOP
                          Exp (9)
                            Exp (9)
                              Exp (9)
                                Exp (9)
                                  Exp (9)
                                    LP
                                    Exp (9)
                                      Exp (9)
                                        NOT
                                        Exp (9)
                                          ID: _dPwd
                                          LP
                                          RP
                                      DIV
                                      Exp (9)
                                        Exp (9)
                                          INT: 15423
                                        DOT
                                        ID: SuPihyf
                                    RP
                                  DIV
                                  Exp (9)
                                    ID: utLNtLS
                                    LP
                                    RP
                                DIV
                                Exp (9)
                                  INT: 508
                              STAR
                              Exp (9)
                                INT: 25010
                            STAR
                            Exp (9)
                              NOT
                              Exp (9)
                                ID: ktrVx5
                                LP
                                Args (9)
                                  Exp (9)
                                    LP
                                    Exp (9)
                                      FLOAT: 2730.179932
                                    RP
                                RP
                      ASSIGNOP
                      Exp (9)
                        Exp (9)
                          Exp (9)
                            FLOAT: 4836776476672.000000
                          LB
                          Exp (9)
                            ID: yiozwUb9Ym
                          RB
                        OR
                        Exp (9)
                          Exp (9)
                            ID: t8BbRFZ0
                          DIV
                          Exp (9)
                            ID: kwnAoY8
                    SEMI
                  StmtList (10)
                    Stmt (10)
                      Exp (10)
                        ID: N_m7wo0d
                      SEMI
                RC
            StmtList (12)
              Stmt (12)
                Exp (12)
                  Exp (12)
                    NOT
                    Exp (12)
                      MINUS
                      Exp (12)
                        ID: ZqLP
                  ASSIGNOP
                  Exp (12)
                    Exp (12)
                      ID: g7e52kv
                    MINUS
                    Exp (12)
                      FLOAT: 0.000008
                SEMI
        RC
    ExtDefList (14)
      ExtDef (14)
        Specifier (14)
          StructSpecifier (14)
            STRUCT
            OptTag (14)
              ID: OM7yrgCX
            LC
            DefList (15)
              Def (15)
                Specifier (15)
                  TYPE: float
                DecList (15)
                  Dec (15)
                    VarDec (15)
                      VarDec (15)
                        ID: AcbtgCZ
                      LB
                      INT: 28132
                      RB
                SEMI
              DefList (16)
                Def (16)
                  Specifier (16)
                    TYPE: float
                  DecList (16)
                    Dec (16)
                      VarDec (16)
                        VarDec (16)
                          ID: UMo7LK8uH
                        LB
                        INT: 2812
                        RB
                      ASSIGNOP
                      Exp (16)
                        ID: lfky7CXb7h8
                        LP
                        Args (16)
                          Exp (16)
                            ID: Gc_P51G
                            LP
                            Args (16)
                              Exp (16)
                                FLOAT: 85.059998
                            RP
                        RP
                  SEMI
                DefList (17)
                  Def (17)
                    Specifier (17)
                      TYPE: float
                    DecList (17)
                      Dec (17)
                        VarDec (17)
                          ID: QOZ7u1F
                    SEMI
            RC
        ExtDecList (19)
          VarDec (19)
            VarDec (19)
              ID: dicB5v8AH
            LB
            INT: 727
            RB
          COMMA
          ExtDecList (19)
            VarDec (19)
              ID: BOM6
            COMMA
            ExtDecList (19)
              VarDec (19)
                ID: l0iaji
              COMMA
              ExtDecList (19)
                VarDec (19)
                  ID: JrnzCC4G8d
                COMMA
                ExtDecList (19)
                  VarDec (19)
                    ID: LJ99NK6Y
                  COMMA
                  ExtDecList (19)
                    VarDec (19)
                      VarDec (19)
                        VarDec (19)
                          ID: cDTCz
                        LB
                        INT: 1223
                        RB
                      LB
                      INT: 3
                      RB
                    COMMA
                    ExtDecList (19)
                      VarDec (19)
                        VarDec (19)
                          ID: AMXKE
                        LB
                        INT: 3287
                        RB
        SEMI
      ExtDefList (20)
        ExtDef (20)
          Specifier (20)
            TYPE: int
          ExtDecList (20)
            VarDec (20)
              ID: NAOUVsQOD
          SEMI
